core-bundle/contao/languages/en/default.xlf

```diff
@@ -284,7 +284,7 @@
       <trans-unit id="ERR.personalDataPassword">
         <source>There is at least one "personal data" module that allows the password to be changed. This is a security weakness and will no longer work in Contao 6. Use the "change password" module instead.</source>
       </trans-unit>
-      <trans-unit id="ERR.copyAllUnique">
+      <trans-unit id="ERR.copyUnique">
```

Comment:
Changing the key forces all translators to translate the string again. Is this really necessary?

Reply:
This is a new translation key, not an existing one.

Reply:
I see. I had the "hide whitespace" option enabled and then it looked like you had changed the key. Never mind.